application/F3DStarter.cxx

```diff
@@ -1636,4 +1639,13 @@ void F3DStarter::AddCommands()
         this->LoadFileGroup(index);
       }
     });
+    interactor.addCommand("open_file", 
+      [this](const std::vector<std::string>& args)
+      {
+         char* file = tinyfd_openFileDialog( "Open File", ".", 0, nullptr, nullptr, false);
+         if (file) {
+           int index = this->AddFile(file);
+           if (index > -1) this->LoadFileGroup(index);
+         }
```

Comment:
we use the following style, always:

```
if ()
{
}
```

Reply:
Should be fixed now, I didn't notice that initially when checking the code style guide.